HW04/Lizaveta_Muromskaya/student.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require './human'
+require './mentor'
+require './homework'
+require './notification'
+
+# This class is responsible for the student's actions
+class Student < Human
+  attr_reader :homeworks, :notifications
+  attr_accessor :mentors
+
+  def initialize(name:, surname:)
+    super
+    @homeworks = []
+    @mentors = []
+    @notifications = []
+  end
+
+  def all_homeworks
+    @homeworks.each { |homework| p homework }
+  end
+
+  def to_work(title, data)
+    @homework = Homework.new(title: title, data: data)
+  end
+
+  def add_answer
+    @homeworks << @homework
+  end
```

Comment:
That sounds strange to me 🤔 Maybe we should change interface a little bit because you didn't change `@homework` at all

For example

```suggestion
  def add_answer(answer)
    @homework.answer = answer
  end
```